ksqldb-functional-tests/src/test/resources/query-validation-tests/joins.json

```diff
@@ -895,9 +895,7 @@
         {"topic": "right_topic", "key": 3, "value": {"L2": "d"}, "timestamp": 60000}
       ],
       "outputs": [
-        {"topic": "LEFT_JOIN", "key": 0, "value": {"L1": "A", "L2": null}, "timestamp": 0},
         {"topic": "LEFT_JOIN", "key": 0, "value": {"L1": "A", "L2": "a"}, "timestamp": 60000},
-        {"topic": "LEFT_JOIN", "key": 1, "value": {"L1": "B", "L2": null}, "timestamp": 330000},
         {"topic": "LEFT_JOIN", "key": 2, "value": {"L1": "C", "L2": null}, "timestamp": 90000},
```

Comment:
Why do we still have this result record?

Reply:
This is part of the issue about grace periods we talked before, where an expired record is still added to the window. The left/outer join, though, has the logic to emit expired records, but after the expired record is still added to the normal join window later.

Reply:
Ah. Good to know. -- Guess we need to fix this in a follow up... But the bug is there since day one, so not a big deal I guess.